easybuild/easyblocks/c/cuda.py

```diff
@@ -132,8 +132,9 @@ def sanity_check_step(self):
         custom_paths = {
             'files': ["bin/%s" % x for x in ["fatbinary", "nvcc", "nvlink", "ptxas"]] + extra_files +
                      ["%s/lib%s.%s" % (x, y, shlib_ext) for x in chk_libdir for y in ["cublas", "cudart", "cufft",
-                                                                                      "curand", "cusparse"]],
-            'dirs': ["include"],
+                                                                                      "curand", "cusparse"]] +
+                     ["extras/CUPTI/%s/lib%s.%s" % (x, y, shlib_ext) for x in chk_libdir for y in ["cupti"]],
```

Comment:
Since there's only one library in the `y` list, you can simplify this to:

```python
["extras/CUPTI/%s/libcupti.%s" % (x, shlib_ext) for x in chk_libdir],

Reply:
Just took a quick look, this needs to be done conditionally based on the CUDA version, it's only there since CUDA 7.0 it seems, and we still have easyconfigs around for CUDA 5.x and 6x.